src/chainstate/burn/operations/stack_stx.rs

```diff
@@ -116,6 +117,9 @@ impl PreStxOp {
             return Err(op_error::InvalidInput);
         };
 
+        let mut memo = tx.data();
+        memo.truncate(77);
```

Comment:
Is there a reason we're limiting it to 77 bytes?

Reply:
Yes, why 77? Should it be 34, like the `stx-transfer-memo?`?

Reply:
Oh, I see the stx transfer burnchain op allows up to 61 bytes, so the size is not consistent.

Reply:
Ah, it's 77 because it's just 2 magic bytes, 1 byte for the op, and the rest are memo.